### PR TITLE
MB-15796: TOO can view review request service item update modal

### PR DESCRIPTION
**Pull request by StuTruss**
## Summary

This PR uses the customizable modal @afoon create [here](https://github.com/transcom/mymove/pull/10648) to create a preliminary modal that shows the `HHG` tag as well as the title `Review request: service item update` because of @afoon changes, the service item details portion is already included even though that was not a part of the AC of the original ticket.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers
- [ ] HHG Label is displayed in top left corner: 
- [x] Title of modal is `Review request: service item update`
- [ ] Has unit tests

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access `http://officelocal:3000/testharness/list` and scroll down to button that says `HHGMoveInSITWithAddressChangeRequestOver50Miles` then click it. 
2. Copy value of `locator` field
3. Access the Office App
4. Login as a TOO
5. Navigate to move with value of `locator` field you copied earlier
6. Click `Move task order` tab
7. Scroll down to `Review Request` button in approved service items portion and click it

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/about/supported-browsers) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots
Review Request Modal
Screenshot 2023-05-22 at 1 42 36 PM

### CODE REVIEW

**Comment by AnnaGingle** [comment]
The review modal looks good but I am not able to check the edit modal because the functionality isn't in place yet.

**Review by afoon — CHANGES_REQUESTED** [review]
This is missing the form element where the TOO can approve or reject the Request depicted in the designs. `ServiceItemUpdateModal` accepts the form as a child.
Screenshot 2023-05-22 at 5 48 34 PM

**Comment by StuTruss** [comment]
@afoon @AnnaGingle maybe I'm misunderstanding the ticket? Because in the implementation notes it says that this ticket should not include the SIT Address Information nor anything about the edit modal.

**Comment by afoon** [comment]
@StuTruss The A/C is confusing me as well. I'm not sure why this story has less A/Cs than the Edit Address Form which also includes the form and functionality.

**Review by AnnaGingle — CHANGES_REQUESTED** [review]
The alert & number tags should not be visible when there is not an active request from the Prime


https://github.com/transcom/mymove/assets/48488699/c6037b68-86ff-4314-a785-afaa7b83d550

File: src/components/Office/ServiceItemDetails/ServiceItemDetails.jsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const ServiceItemDetails = ({ id, code, details }) => {
                     secondCustomerContact && secondCustomerContact.timeMilitary
                       ? secondCustomerContact.timeMilitary
                       : '-',
-                  'First available delivery date 2':
+                  'Available delivery date 2':
```
Comment (afoon) [inline src/components/Office/ServiceItemDetails/ServiceItemDetails.jsx:69]: Same revert here.
 ```suggestion
                  'First available delivery date 2':
```

**Review by afoon — CHANGES_REQUESTED** [review]
I'm going to hold off on reviewing `MoveTaskOrder.jsx` and `ServiceItemsTable.jsx` until the most recent changes from the base branch are pulled into this PR. Some new patterns were introduced that will affect your work.

But I did want to flag reverting the label changes in ServiceItemDetails. I added more context below.

**Review by AnnaGingle — CHANGES_REQUESTED** [review]
- Office Remarks field should be 136px in height
- Missing border & additional padding around Final SIT delivery content
- Missing UPDATE REQUESTED tag
Anna's private board - 10728

**Comment by afoon** [comment]
I changed the base branch to make it easier to see what changes were made just on this PR. When the Edit Address Modal PR is merge into main, Github will automatically base it back to main again.

Additionally, if ReviewRequestServiceItem work is complete before Edit Modal is ready to merge, we can merge the ReviewRequestModal changes into Edit Address Modal branch. Then changes for both PRs will be merged into main at the same time.

cc: @rogeruiz @StuTruss @donaldthai

**Review by AnnaGingle — DISMISSED** [review]
Change 'Final SIT delivery' label to 'Final SIT delivery address'
Designs: 
Edit SIT destination address change - onClick

**Review by AnnaGingle — APPROVED** [review]
Great work team!

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

Good work on this @StuTruss.

**Comment by rogeruiz** [comment]
I've gotten confused by the two PRs, so I'm going to merge this into @afoon's PR so I'm working from a single branch.

**Pull request merged**